scripts/globals/mobskills/self-destruct_1.lua

```diff
@@ -17,7 +17,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local damage = skill:getMobHP()
+    local damage = skill:getMobHP() / 3
```

Comment:
Where are you getting the values for the self destruct damage from? Just for future reference.

Reply:
It would be the same formula as a "regular" self-destruct. As the first cluster will go off between 66-99% HP - compared to going off 1-33%. These fractions maintain a constant damage throughout.

Where say a regular bomb had 30% HP, 30 / 3 would be a 10 dmg mod. Whereas in a cluster, at 90% HP, 90 / 9 would continue the similar 10 dmg mod.

Reports of clusters were dealing 2k dmg as the dmg mod was set straight to the mob's HPP (66-99)

Reply:
Albeit this can be temporary until better data presents itself. This prevents clusters from being untouchable.

Reply:
https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=57955395

Reply:
Take this with a grain of salt as I'm not too familiar with the codebase (_yet_), but from what I can see...

If I'm reading this right, it looks like this only affects the damage of the Self-Destruct, but not the HP threshold at which it occurs or the frequency of multiple bombs detonating at once?
I'm not sure if this PR is intended to address the entire issue, but if so then there were three items mentioned in the original issue #2684 and only the first one appears to be modified here:

**1. Damage of self-destruct was too high**
2. Self-destruct was occurring above the 66% threshold
3. 2-bomb and 3-bomb destructs are occurring above the 20% HP threshold

The sheet linked above indicates that self-destruct can't occur until 66%, and this is supported by historical wiki information
Regarding 2-bomb and 3-bomb detonations, the earliest mention I can find of that is from August 2006 but it indicates that detonating multiple bombs at once is a chance _only_ when the mob is below 20% HP:
https://ffxiclopedia.fandom.com/wiki/Category:Clusters?oldid=103834

However, if I'm understanding the code correctly (both before and after the proposed changes) it looks like it's just a flat 20% chance to do a full-detonate at _all_ HP values, and a self-destruct can also occur at all HP values as well.